### DISCUSSION TITLE
Version 3.0.0 Build 104

### DISCUSSION THREAD

**Discussion by SMartorelli** [IBM/mac-ibm-notifications#168] [General]
### Improvements
* Migrated the UI from AppKit to SwiftUI for modernized UI experience and streamlined feature development. #110 
* Integrated SF Symbol support for icons. #113 
* Added `-background_panel` argument for defining a comprehensive background view across all screens/spaces. #89
* Introduced `-unmovable` argument to ensure the popup/onboarding UI remains stationary on the screen.  #152
* Introduced a new `datepicker` accessory view. #119
### Bug Fixes
* Resolved a bug causing incorrect interpretation of / in progress bar payloads.
* Fixed an issue where lengthy text would be cut off at the end when used with the checklist accessory view.
* Corrected a problem that returned the wrong code when the cmd+q keyboard shortcut was used. #138

### Improvements
* Implemented a tracking feature for the "Play user interface sound effect" setting to prevent IBM Notifier from playing pop-up sounds when the option is disabled.
### Bug Fixes
* Resolved a bug causing wrong representation of the pop-up text when custom size of the icon were defined. #143

### Improvements
* Enabled defining a Tertiary button on the onboarding page. #123
* Added the capability to set custom labels for onboarding buttons. #131 
* Integrated a feature to set a timeout for the onboarding UI. #154
* Developed a new onboarding progress bar mode to track the progress of onboarding UI. #109
### Deprecated
* Markdown support for Onboarding UI's subtitle and body texts on macOS 11

## Notes to the release
* We have updated our deployment target to macOS 11 Big Sur. This decision was made due to technological constraints with SwiftUI present in macOS 10.15 and earlier versions.

<hr /><em>This discussion was created from the release <a href='https://github.com/IBM/mac-ibm-notifications/releases/tag/v-3.0.0-b-104'>Version 3.0.0 Build 104</a>.</em>

**Comment by jbmaruh** [comment]
Want to note that the `-panel_background` is actually `-background_panel` within the code. Once corrected to the proper argument both the opaque and translucent verbs work great.

**Reply by SMartorelli** [reply]
Thanks for the report. I’ve fixed the typo in the release CL 🙏

**Comment by kryptonit1** [comment]
Some onboarding UI feedback:

I noticed that the spacing between the **topIcon** and the **title** seems to have decreased compared to 2.9.1. I don’t now if it’s due to fonts being rendered differently with SwiftUI compared to AppKit or if the actual spacing has been decreased? The font size of both the title and subtitle seems to have been decreased as well. It would probably be a good idea to review the font sizes. To give an example: the size of the subtitle currently seems to be the same as the body. I think I like the font sizes in 2.9.1 better.

![Screenshot 2023-08-01 at 11 45 55](https://github.com/IBM/mac-ibm-notifications/assets/33235315/d2f511f9-9742-4fd6-a734-2fbfe5e21697)
![Screenshot 2023-08-01 at 11 47 57](https://github.com/IBM/mac-ibm-notifications/assets/33235315/64f5424d-83ce-4210-a085-930522f6dce0)
![Screenshot 2023-08-01 at 12 00 56](https://github.com/IBM/mac-ibm-notifications/assets/33235315/cd51325f-d0d4-40fd-b2c9-679bb8f09d09)

**Reply by kryptonit1** [reply]
Have changes been made when it comes to how HTML is being rendered in 3.0.0? Please see comparison below.

**2.9.1**
![html_ibm_notifier_2 9 1](https://github.com/IBM/mac-ibm-notifications/assets/33235315/87eaf432-5014-4d67-b41c-f2d863b9b799)

**3.0.0**
![html_ibm_notifier_3 0 0](https://github.com/IBM/mac-ibm-notifications/assets/33235315/710f3018-1a4a-45b4-b3df-fc7b0b9e6680)

I also miss the divider/border at the bottom of the onboarding window (between the window content and the buttons):
![divider_ibm_notifier](https://github.com/IBM/mac-ibm-notifications/assets/33235315/86656702-cc6e-413f-bd97-f682f8df9f4e)

**Reply by SMartorelli** [reply]
Will work to fix this too in 3.0.1

**Reply by SMartorelli** [reply]
@kryptonit1 please check the latest release. About the subtitle style remember that it supports markdown so you can use any related title style `### Title 3, ## Title 2, # Title 1`
